_guides/project-boards/configure.md

```diff
@@ -256,11 +256,13 @@ However, you also have complete control to change how each card is displayed.
 
 To demonstrate this, let's make the cards for closed tasks more obvious.  We don't need to see all those fields once something is completed.
 
-1. Click the gear button (⚙️) at the top of the project board's profile page.
+1. Navigate to the project board's profile page by clicking the search feature and then selecting project boards.
+1. Then select the desired project board you wish to work on.
```

Comment:
I'd refrain from starting list items with "then" or "next". It's already a numbered list, so that's inherent.